adminpages/wizard/done.php

```diff
@@ -0,0 +1,90 @@
+<?php
+	$site_type = get_option( 'pmpro_site_type', true );
+
+	if ( empty( $site_type ) ) {
+		$site_type = __( 'general', 'paid-memberships-pro' );
```

Comment:
```suggestion
		$site_type = 'general';
```
This is meant to be saved in the database, not seen by the user.